sdk/python/feast/feature_table.py

```diff
@@ -333,12 +335,12 @@ def from_proto(cls, feature_table_proto: FeatureTableProto):
             batch_source=(
                 None
                 if not feature_table_proto.spec.batch_source.ByteSize()
```

Comment:
What ByteSize?

Reply:
Removed for batch_source, only kept for stream_source since it's optional.